virttest/utils_qemu.py

```diff
@@ -160,3 +162,19 @@ def get_maxcpus_hard_limit(bin_path, machine_type):
     except KeyError:
         raise ValueError("Could not get the maximum limit CPUs supported by "
                          "this machine '%s'" % machine_type)
+
+
+def get_dev_attrs(opt_name, dev_name, params):
```

Comment:
```suggestion
def get_dev_attrs(bin_path, opt_name, dev_name):
```
According others function design and from this module side, this module provides the APIs should be as a basic function for caller, so we better to make sure the type of argument of function should be built-in type.
And this function mainly to get the device attributes by a qemu_binary provided. So I think we don't need to do external work for getting qemu binary, but assign it for caller to do that.